On prem installation revamp (#2164)
mendersoftware/mender-docs: 07.Server-installation/01.Overview/docs.md

    category: docs
---

> !!! For evaluation of the on premise installation of the Mender Enterprise Server please [contact us](https://mender.io/contact-us). In the message please mention the 'Evaluation of the on-prem Enterprise Mender'.

Reviewer: ```suggestion
!!! For evaluation of the on-premise installation of the Mender Enterprise Server please [contact us](https://mender.io/contact-us). In the message please mention'Evaluation of the on-prem Enterprise Mender'.
```

Mender backend is composed of a number of microservices, each implementing a

> small, well defined piece of functionality. Services are delivered in form of Docker images, available from official [Mender Docker repository](https://hub.docker.com/r/mendersoftware/?target=_blank). When required, each service can be built directly from its source code.

Reviewer: ```suggestion
small, well defined piece of functionality. Services are delivered as Docker images, available from the official [Mender Docker repository](https://hub.docker.com/r/mendersoftware/?target=_blank). When required, you can build each service directly from its source code.
```

## Chapter structure

> For the quickest way to a working local server with default configuration and keys, please read [Evaluation with Docker compose](../02.Evaluation-with-docker-compose/docs.md). By following a list of copy-pastable steps you will end with a working server environment of your computer.

Reviewer: ```suggestion
For the quickest way to a working local server with the default configuration and keys, please read [Evaluation with Docker compose](../02.Evaluation-with-docker-compose/docs.md). By following a list of copy-pastable steps you will end with a working server environment of your computer.
```

> On a running mender server instance the microservices communicate together and have been set up with the correct configurations and keys. To know more about the details on the architecture and configuration please read the [Architecture](01.Architecture/docs.md) and [Certificates and keys](02.Certificates-and-keys/docs.md).

Reviewer: ```suggestion
On a running mender server instance, the microservices communicate together and have been set up with the correct configurations and keys. To know more about the details on the architecture and the configuration please read the [Architecture](01.Architecture/docs.md) and [Certificates and keys](02.Certificates-and-keys/docs.md).
```

> For production installations please follow the setup described in [Production installation with Kubernetes](../04.Production-installation-with-kubernetes/docs.md). It describes how to set up a scalable mender server installation in on of the big cloud providers.

Reviewer: ```suggestion
For production installations, please follow the setup described in [Production installation with Kubernetes](../04.Production-installation-with-kubernetes/docs.md). It explains how to install a scalable mender server in one of the big cloud providers.
```

## Support

[Support](https://mender.io/support) covered by the Enterprise plan is valid for:

> * production versions installed with kubernetes

Reviewer: ```suggestion
* production versions installed with Kubernetes
```

* component versions as defined under [Production installation with Kubernetes](../04.Production-installation-with-kubernetes/docs.md)

> Following the above points will ensure a simplest working production installation regardless of the Opens Source or Enterprise server.

Reviewer: ```suggestion
The above points ensure a straightforward working production installation regardless of the Opens Source or Enterprise server.
```

> !!!!! Please note that the Mender Server containers comes released as part of the [Mender Product bundle](../../302.Release-information/01.Release-schedule/docs.md) and falls under the support schedule explained there.

Reviewer: ```suggestion
!!!!! Please note that the Mender Server containers come released as part of the [Mender Product bundle](../../302.Release-information/01.Release-schedule/docs.md) and fall under the support schedule explained there.
```
